src/main/java/it/pagopa/pn/delivery/models/InputSearchNotificationDto.java

```diff
@@ -46,15 +46,17 @@ public class InputSearchNotificationDto {
     
     private final boolean bySender;
 
+    private final boolean isPrivate;
```

Comment:
Non mi piace molto questo isPrivate nel Dto. E' fuori contesto.
Non si può riutilizzare il codice in altro modo?

Reply:
Ok